Dockerfile

```diff
@@ -22,4 +22,5 @@ RUN curl "https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_V
 ENV GOPATH /gopath
 ENV PATH "${PATH}:${GOPATH}/bin"
 RUN go get -u github.com/golang/lint/golint
+RUN go get -u github.com/ghodss/yaml
```

Comment:
this feels like all this review is about. If so could you rebase onto master and make this the only change? or if it is easier just start a new branch/PR with this being the only change?